rplugin/python3/deoplete/source/around.py

```diff
@@ -0,0 +1,57 @@
+import re
+from .base import Base
+
+from deoplete.logger import getLogger
+from deoplete.util import parse_buffer_pattern
+
+logger = getLogger('around')
+
+
+class Source(Base):
+    def __init__(self, vim):
+        super().__init__(vim)
+        self.name = 'around'
+        self.mark = '[~]'
+        self.rank = 800
+
+    def gather_candidates(self, context):
+        candidates = list()
+
+        # 20 lines above
+        lnum = context['position'][1]
+        words = parse_buffer_pattern(
+                        reversed(self.vim.call('getline',
+                                               max([1, lnum - 20]),
```

Comment:
It would be better to be a configurable. At least a constant variable should be used.